.github/workflows/validate-snapshots.yml

```
@@ -0,0 +1,74 @@
+
+jobs:
+  validate-snapshots:
+    name: Validate Test Snapshots
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)